.travis.yml

```diff
@@ -29,16 +27,68 @@ env:
     - secure: Lrh6AxUp1yFdsB1XlvNHFk+5pOMGQR8bJadUtnqXsuk97ieINI5w/da6wBPIH4WkZBGpQr+uJLe10PTxDcaNeZOC1LcuSK+v5J88pSiFYi10BggYWC2Kb2XEVBjd9BHi/RxAkiHXziBEaqpuX2WcwY2hgAOU+rpt3M9U9SxDITXNG68PpVVqDJ7GSZvZeJZnLrjTPKLlFR8EkqrE23bTaGvzqn50fwk7U3n0F04slkmg2ORScWkgBSFmaATBbiSU9Pz7bB4KuGarNC0Q0eTeoZcThRVYVwse3oLnYLMXkUuIDF5QwDfA5HCC4ad5yXdVenwHZSPS/sSnRM+fVAwkU8CL7hFC9zX2X/fBExWet0LufaCh4L6pax3dKV2m6q3Uq9M1z8X33ry2xqS96rpdD4bo+FxYrzjyAotn46w4fPE7yb3YZgf+dbHTDfVjiPH70hIKUhKiyzyKMTwdT63WNIy1/qbREge4jPVXgnIM3/jAu0THf74cT+RwN4xQnt7a50Yn2c/I+37kO2B+4qkyh4p75eIfPXqak1vdDc9tUqxZa7lPdIkofT3jp7TaJ+4/OLf2OrpA2XFjx20el4/TdB9TiJtGv0qTrujY4wh5BALxGVoSEihj3ahkqV/FQ6RyWJsMggd3yoaING61nCkDXUvLk/44yzeYavNrUaMzWpc=
 jobs:
   include:
-    - stage: "Verify AWSS3"
+    - stage: "Verify Packages"
+      name: AWSS3
       julia: 1.0
       os: linux
       script:
         - julia --project=tmps3 -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.add("AWSS3"); Pkg.test("AWSS3")'
-    - stage: "Verify AWSSQS"
+    - stage: "Verify Packages"
+      name: AWSSQS
       julia: 1.0
       os: linux
       script:
         - julia --project=tmpsqs -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.add("AWSSQS"); Pkg.test("AWSSQS")'
+    - stage: "EC2"
+      julia: 1.0
+      os: linux
+      before_install:
+        - pip install --user awscli
+      script:
+        - aws ssm get-parameter --name /public_ci/AWSCore/ec2_keyfile --query 'Parameter.Value' --output text --with-decryption > ec2.pem
+        - chmod 600 ec2.pem
+        - export KEYFILE=ec2.pem
+        - STACKNAME="awscore-$TRAVIS_COMMIT"
+        - aws cloudformation create-stack --stack-name $STACKNAME --template-body file://test/aws/ec2.yml --capabilities CAPABILITY_IAM
+        - aws cloudformation wait stack-create-complete --stack-name $STACKNAME
+        - export EC2Ip=$(aws cloudformation describe-stacks --stack-name $STACKNAME --query 'Stacks[0].Outputs[?OutputKey==`EC2Ip`].OutputValue' --output text)
+        - scp -i $KEYFILE -o "StrictHostKeyChecking=no" -r . ec2-user@$EC2Ip:AWSCore
+        # Fix `MethodError: no method matching getindex(::Nothing, ::String)` when installed on a fresh stack
+        - |
+          julia -e '
+          using Distributed
+          addprocs(
+              ["ec2-user@$(ENV["EC2Ip"])"],
+              sshflags=`-i $(ENV["KEYFILE"]) -o "StrictHostKeyChecking=no"`,
+              exename="/julia/bin/julia",
+              dir="/home/ec2-user/AWSCore",
+              tunnel=true,
+          )
+          @everywhere using Pkg
```

Comment:
Instead of `@everywhere` you can get the worker id back from `addprocs` and use `@spawnat $worker_id using Pkg` etc.

Reply:
It seems both `@spawn` and `@spawnat` don't like "using". 

```julia
@spawnat $worker_id using Pkg
ERROR: syntax: "using" expression not at top level
```

Reply:
I think it would work with `@spawnat 2 @eval using Pkg`